### ISSUE TITLE
When is the next release? 

### ISSUE THREAD

**Issue by ashygk** [surveyjs/widgets#75]
Hi, There were fixes included we need for the next release. May I know when it is going to be released? I don't see a release happened this tuesday, i.e. Aug 14th, as you guys generally release every Tuesday.

**Comment by andrewtelnov** [comment]
@ashygk You are right. We did not release this week. We are going to make a release on Monday-Tuesday next week. Do you need this release urgently?

Thank you,
Andrew

**Comment by ashygk** [comment]
@andrewtelnov Thanks for getting back. Yes, we were waiting for the release.

**Comment by andrewtelnov** [comment]
@ashygk How bad are you waiting it? I am the only person in the office for today and tomorrow, vacation time :)
Could it wait till Monday, or do you need it badly?

Thank you,
Andrew

**Comment by ashygk** [comment]
Our last sprint failed because of the updates we were waiting for. We are in the new sprint now and still waiting to close them after receiving the update. Could wait till Monday, but would appreciate if available at the earliest. Thanks :)

**Comment by andrewtelnov** [comment]
@ashygk Got it.

Thank you,
Andrew

**Comment by andrewtelnov** [comment]
@ashygk v1.0.38 is available.

Thank you,
Andrew

**Comment by ashygk** [comment]
@andrewtelnov Thank you so much!

**Comment by ashygk** [comment]
@andrewtelnov  Could you please re-open issue #1259 in surveyjs issues . The issue has been closed without verifying and the fix doesn't work
